src/repos/ArticleRepo.ts

```diff
@@ -43,61 +43,65 @@ const getAllArticles = async (
   limit = DEFAULT_LIMIT,
 ): Promise<Article[]> => {
   return ArticleModel.find({})
+    .sort({ date: 'desc' })
     .skip(offset)
     .limit(limit)
     .then((articles) => {
       return articles.filter((article) => !isArticleFiltered(article));
     });
 };
 
-const getArticlesByPublicationID = async (
-  publicationID: string,
+const getArticlesByPublicationSlug = async (
+  slug: string,
   limit: number = DEFAULT_LIMIT,
   offset: number = DEFAULT_OFFSET,
 ): Promise<Article[]> => {
-  const publication = await (await PublicationModel.findById(publicationID)).execPopulate();
-  return ArticleModel.find({ 'publication.slug': publication.slug })
+  return ArticleModel.find({ 'publication.slug': slug })
+    .sort({ date: 'desc' })
     .skip(offset)
-    .limit(limit)
-    .then((articles) => {
-      return articles.filter((article) => !isArticleFiltered(article));
```

Comment:
what happened to returning the filtered articles?

Reply:
I see we still have this for the publicationID - should we not also have a filter on the slugs? To prevent being blocked on the App Store?